streams/src/main/java/org/apache/kafka/streams/kstream/ValueTransformer.java

```diff
@@ -77,9 +75,6 @@ public interface ValueTransformer<V, VR> {
 
     /**
      * Transform the given value to a new value.
-     * Additionally, any {@link StateStore} that is {@link KStream#transformValues(ValueTransformerSupplier, String...)
-     * attached} to this operator can be accessed and modified arbitrarily (cf.
-     * {@link ProcessorContext#getStateStore(String)}).
```

Comment:
Should we keep this comment and link to `KTable#transformValues(...)` instead of `KStream` ?

Reply:
Great idea!